.code-samples.meilisearch.yaml

```diff
@@ -0,0 +1,38 @@
+landing_getting_started_1: |-
+    # Configure your entities by adding them in config/packages/meili_search.yaml
```

Comment:
Since this file has two different languages, I must use markdown here, or the landing page will understand it? 

CC: @bidoubiwa @mdubus